apps/condo/domains/ticket/schema/TicketReportService.test.js

```diff
@@ -17,9 +17,11 @@ describe('TicketReportService', () => {
             })
             expect(data).toBeInstanceOf(Array)
             expect(data.length).toBeGreaterThanOrEqual(1)
-            expect(data[0]).toHaveProperty('statusName')
-            expect(data[0]).toHaveProperty('currentValue')
-            expect(data[0]).toHaveProperty('growth')
+            expect(data.find(e => e.statusName === 'Открыта')).toMatchObject({
```

Comment:
pleas use status type instead of it name

Reply:
@pahaz status type not used in ui component